integration-tests/ssr/__tests__/ssr.js

```diff
@@ -33,7 +33,7 @@ describe(`SSR`, () => {
     expect(String(childProcess.stdout)).toContain(
       `testing these paths for differences between dev & prod outputs`
     )
-  }, 30000)
+  }, 60000)
```

Comment:
Let's leave the changes to SSR test out of this and put it into a separate PR :)